tensorflow_lite_support/python/task/audio/core/audio_record_test.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for audio_record."""
+
+import unittest
+import numpy as np
+import sounddevice as sd
+
+from numpy.testing import assert_almost_equal
+from absl.testing import parameterized
+
+from tensorflow_lite_support.python.task.audio.core import audio_record
+
+
+class MockInputStream(sd.InputStream):
+  def __init__(self, input_data, callback=None, **kwargs):
+    def audio_callback(data, *_):
+      return callback(input_data, *_)
+    super().__init__(callback=audio_callback, **kwargs)
+
+
+class AudioRecordTest(parameterized.TestCase, unittest.TestCase):
+  def setUp(self):
```

Comment:
@khanhlvg I've added a new test. Should this be fine?

Reply:
I think the module still expects an input device to be present regardless of patching InputStream's init or not.